### ISSUE TITLE
Mouse not constrained to screen

### ISSUE THREAD

**Issue by jukibom** [jukibom/FlyDangerous#171]
Bug report, I don't know how this one is possible but hey ho

Edit: need to look into this more. It definitely impacts VR but also potentially flat screen? See discussion below.

**Comment by jukibom** [comment]
Can't reproduce this one, any help anyone?

**Comment by jukibom** [comment]
going to assume this is in VR, which is definitely a problem

**Comment by triayak** [comment]
Hey, 
i didnt initially report this, but experienced it yesterday during time trials while using mouse control. Was slightly annoying since the mouse was going crazy on the second monitor, but i didnt think much of it until i just read this issue randomly :) I cant seem to reproduce it right now though, will follow up if i do. 

Love the game! Keep being awesome!

**Comment by triayak** [comment]
can definitely reproduce it now, happens almost all the time. I feel it happens when i stop holding down left click (=boost). If i keep holding boost for a full level i feel like im fine. After the mouse escapes clicking sometimes still boosts and sometimes klicks on the second monitor. Pls let me know if i can help with providing logs or smth.

**Comment by jukibom** [comment]
So this is Flat Screen (not VR)? What platform?

**Comment by triayak** [comment]
Yes! Sorry, i didnt clarify that. Platform is Win 10 through Steam, standard game version, no beta.

**Comment by jukibom** [comment]
Don't suppose you happen to know if it occurs in beta do you?

**Comment by triayak** [comment]
not yet, will try :)

**Comment by roncli** [comment]
I'm having this issue as well.  I don't know if it's related, but I use LMB for left thruster and RMB for right thruster, and seems to happen when I'm using left thrusters.

**Comment by jukibom** [comment]
> I'm having this issue as well. I don't know if it's related, but I use LMB for left thruster and RMB for right thruster, and seems to happen when I'm using left thrusters.

Interesting, will take another look!
